samples/azure/performance-tuning-asb/ASB_7/Shared/ReceiveCounter.cs

```diff
@@ -1,23 +1,34 @@
 ﻿using System;
-using System.Reactive.Linq;
-using System.Reactive.Subjects;
+using System.Threading;
+using System.Timers;
```

Comment:
my understanding is System.Timers.Timer is primarily for a design surface. so in this case we should use System.Threading.Timers. System.Threading.Timers is what we use in the core